src/bucklescript/output_bucklescript_decoder.re

```diff
@@ -189,28 +189,26 @@ let rec generate_decoder = config => {
     )
   | Res_solo_fragment_spread(loc, name) =>
     generate_solo_fragment_spread(conv_loc(loc), name)
-  | Res_error(loc, message) => generate_error(conv_loc(loc), message);
-}
+  | Res_error(loc, message) => generate_error(conv_loc(loc), message)
 and generate_nullable_decoder = (config, loc, inner) =>
   [@metaloc loc]
-  {
+  (
     switch%expr (Js.Json.decodeNull(value)) {
     | None => Some([%e generate_decoder(config, inner)])
     | Some(_) => None
-    };
-  }
+    }
+  )
 and generate_array_decoder = (config, loc, inner) =>
   [@metaloc loc]
-  {
-    %expr
+  [%expr
     value
     |> Js.Json.decodeArray
     |> Js.Option.getExn
-    |> Js.Array.map(value => {
+    |> Js.Array.map(value =>
```

Comment:
It seems compiles don't like this change. Could you please revert it?

Reply:
Yes. I ran it through `refmt`, perhaps a different version? It looks like I didn't change anything in this file, so it was just the changes by `refmt` so I reverted the whole file to the original state.